zookeeper-server/src/main/java/org/apache/zookeeper/server/PrepRequestProcessor.java

```diff
@@ -419,7 +419,7 @@ protected void pRequest2Txn(int type, long zxid, Request request, Record record,
             try {
                 lzks = (LeaderZooKeeperServer) zks;
             } catch (ClassCastException e) {
-                // standalone mode - reconfiguration currently not supported
+                LOG.error("Reconfig operation requested but this feature is currently not supported on standalone mode.”);
                 throw new KeeperException.UnimplementedException();
             }
```

Comment:
- Adding  more logs may be better, I wonder `warn` is better than `error` ?
- @rodrigongarcia It's better to create a [JIRA](https://issues.apache.org/jira/projects/ZOOKEEPER/) issues (sign up JIRA if you don't have an account) to 
bind this PR to a JIRA-ID. The contributor guideline
 is [here](https://cwiki.apache.org/confluence/display/ZOOKEEPER/HowToContribute)

Reply:
I agree with @maoling, I also feel that a WARN would be more appropriate here.